bool    onOFF
float32 depth
float32 depth_der
float32 depth_demand
float32 pitch
float32 pitch_demand
float32 dT0
float32 dT1
float32 T0
float32 T1
float32 thruster0
float32 thruster1
float32 gain
float32 Np
float32 Nc
float32 delta_t
float32 calc_time
float32 km
float32 xf1
float32 xf2
float32 xf3
float32 xf4
float32 xf5
float32 xf6
